### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by NateBigStone**

### CODE REVIEW

File: lmn/templates/lmn/artists/artist_list.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h2 id='artist-list-title'>Artists matching '{{ search_term }}'
     <a href="{% url 'artist_list' %}" id='clear_search'>(clear)</a>
   </h2>
 {% else %}
-  <h2 id='artist-list-title'>All artists</h2>
+  <h2 id='artist-list-title-all'>All artists</h2>
```
Comment (yp0975eu) [inline lmn/templates/lmn/artists/artist_list.html:18]: Is this change necessary? I don't see any CSS or JS related to the ID `artist-list-title-all`

**Review by yp0975eu — APPROVED** [review]
Search works. Tests work. 👍🏼. Tests look good to. A lot of white space changes :/

File: lmn/templates/lmn/artists/artist_list.html (hunk 1 above)
Comment (NateBigStone) [reply lmn/templates/lmn/artists/artist_list.html:18]: @yp0975eu I'm guessing those IDs were intended for selenium since that's how Clara was doing those tests in one of our labs.
I initially changed it to get intellij to stop yelling at me about having duplicate IDs. 
In this case, they shouldn't both render so it's fine.

I'll remove this change. Even if it's necessary it's out of scope. Good catch.

**Comment by NateBigStone** [comment]
> 
> 
> Search works. Tests work. 👍🏼. Tests look good to. A lot of white space changes :/

Thanks for having a look

**Review by sunnycake — APPROVED** [review]
Tests all passed and server ran great with no issues. Was able to search user's own notes.
